core/src/main/java/com/datastax/oss/driver/api/core/cql/CqlSessionBuilder.java

```diff
@@ -13,16 +13,16 @@
  * See the License for the specific language governing permissions and
```

Comment:
`CqlSession` used to be in the `cql` package, but since it's now the main entry point to the driver, I thought it would be more appropriate for it to be in the root package. I should at least add an implementation comment to explain that.
As for `CqlSessionBuilder`, it's true that it seems a bit weird to keep it in another package, I don't have strong feelings either way.